index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+
+		<title>ToDo List</title>
+		<meta name="description" content="ToDo application" />
+		<meta name="author" content="Bartlomiej Zdybel" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+		<link rel="stylesheet" type="text/css" href="./index.css" />
+
+		<link rel="icon" href="icon_code.jpg" />
+	</head>
+
+	<body>
+		<div class="container">
+			<div class="task-list__container">
+				<form class="task-form task-form__add">
```

Comment:
In BEM you need to decide whether given element is a block or an element - either `task-form` or `task-form__add`.